pkg/arvo/sys/zuse.hoon

```diff
@@ -1723,6 +1723,7 @@
           [$deal p=sock q=term r=deal]                  ::  full transmission
           [%goad force=? agent=(unit dude)]             ::  rebuild agent(s)
           [%sear =ship]                                 ::  clear pending queues
+          [%fade dap=term style=?(%gone %doze %wake)]   ::  put app to sleep
```

Comment:
I'd use something other than %wake here, since Behn already uses that.  %jolt, maybe?

Reply:
You're right these are too close to the other meanings of `%doze` and `%wait`.  I've changed them as you suggest, except I've left the user-visible names, since I think for that the original names make more sense.